include/flatbuffers/util.h

```diff
@@ -472,6 +472,12 @@ inline std::string StripFileName(const std::string &filepath) {
   return i != std::string::npos ? filepath.substr(0, i) : "";
 }
 
+// Check if `path` is an absolute path.
+inline bool IsAbsPath(const std::string &path) {
+  return !path.empty() &&
+         (path[0] == kPathSeparator || path[0] == kPathSeparatorWindows);
```

Comment:
Windows abs paths do not start with `\` (though network paths do), they start with `[a-zA-Z]:` .. though not sure if we want to parse all that stuff here, especially if `GetFullPathNameA` works with both. Why was this change needed? If it is an optimisation I'd say leave it out, or if only `realpath` (or `GetFullPathNameA`) don't work correctly with paths that are already absolute, I'd say move it to where they are called.

Reply:
Hm... I thought (if think I ve read that somewhere) internally the paths are posix? So they are not.